examples/performance.jl

```diff
@@ -161,7 +162,7 @@ for (name, kernel) in (
             # compile kernel
             kernel(input, output, Val(Int(bank)), ndrange=size(output))
             for rep in 1:nreps
-                kernel(input, output, Val(Int(bank)), ndrange=size(output))
+                kernel(output, input, Val(Int(bank)), ndrange=size(output))
```

Comment:
This order needs to be changed everywhere

Reply:
Oh, I forgot to change it everywhere... I guess because I just pushed it from GH editor... fixing it fast.